rt/bat/RUNRTQSAM.BAT

```diff
@@ -48,7 +48,7 @@ CALL BAT\ASMLG QSAM\CHKFT.MLC TRACE || goto error
 SET SYSUT2=QSAM\TESTFTL.TFV
 CALL BAT\ASMLG QSAM\BLDFTL.MLC TRACE || goto error
 SET SYSUT1=QSAM\TESTFTL.TFV
-CALL BAT\ASMLG QSAM\CHKFTL.MLC TRACE || goto error
+CALL BAT\ASMLG QSAM\CHKFTL.MLC TRACE TIME(30) || goto error
 DIR QSAM\TEST*.TFV
 
 goto:eof
```

Comment:
You should probably add the ability to add NOTIME parameter to the BLD* steps. See bash/runrtqsam.

Reply:
I have changed both bash and bat versions to accept parameter to bld runs. Also updated the build scripts to use TIME(30) due to the BLDFTL occasionally timing out on GitHub build. Have seen on my own workstation as well.
See 1d6b6cd for commit details.

Reply:
Oops.. didn't test windows.. put a $ instead of a % for variable.. Fixed in f63065e